yash.md

```diff
@@ -2,7 +2,85 @@
 
 _A combination index, glossary, and high level guided tour for the Stanford Digital Repository_
 
-* Access Rights for digital content
+Intended Audience: infrastructure team developers
+
+## Argo
+_The administrative discovery interface for Stanford's Digital Object Registry_
+- ask Andrew, John, JCoyne
+  * Specially chosen staff users (many of them accessioneers)
+
+## infrastructure-integration-test
+_A collection of Capybara integration tests for the SDR_
+- ask any infrastructure dev
+
+## Types of Digital objects
+_We started with more, but we're kinda down to three_
+- ask Andrew
+  * APO
+  * Collection
+  * Item
+
+## cocina-models
+_SDR data model written as syntactically validatable with dry-struct, dry-types and openapi_
+- ask JCoyne or JLitt
+
+## dor-services-app
```

Comment:
Is this duplicative of https://github.com/sul-dlss/infrastructure-team/blob/gh-pages/sdr/index.md#dor-services-app-dsa ?